vnext/Microsoft.ReactNative/ReactNativeAppBuilder.cpp

```diff
@@ -38,29 +43,69 @@ winrt::ReactNative::ReactNativeAppBuilder ReactNativeAppBuilder::SetReactInstanc
 
 winrt::ReactNative::ReactNativeAppBuilder ReactNativeAppBuilder::SetCompositor(
     winrt::Microsoft::UI::Composition::Compositor const &compositor) {
-  m_compositor = compositor;
-
+  m_reactNativeWin32App.as<implementation::ReactNativeWin32App>().get()->Compositor(compositor);
   return *this;
 }
 
 winrt::ReactNative::ReactNativeAppBuilder ReactNativeAppBuilder::SetAppWindow(
     winrt::Microsoft::UI::Windowing::AppWindow const &appWindow) {
-  m_appWindow = appWindow;
+  m_reactNativeWin32App.as<implementation::ReactNativeWin32App>().get()->AppWindow(appWindow);
+
+  return *this;
+}
+
+winrt::Microsoft::ReactNative::ReactNativeAppBuilder ReactNativeAppBuilder::SetReactViewOptions(
+    winrt::Microsoft::ReactNative::ReactViewOptions const &reactViewOptions) {
+  m_reactViewOptions = reactViewOptions;
 
   return *this;
 }
 
-winrt::ReactNative::ReactNativeHost ReactNativeAppBuilder::Start() {
-  auto hwnd{winrt::UI::GetWindowFromWindowId(m_appWindow.Id())};
+winrt::ReactNative::ReactNativeWin32App ReactNativeAppBuilder::Build() {
+  if (m_reactNativeWin32App.Compositor() == nullptr) {
+    m_reactNativeWin32App.as<implementation::ReactNativeWin32App>().get()->Compositor(nullptr);
+  }
+
+  if (m_reactNativeWin32App.AppWindow() == nullptr) {
+    auto appWindow = winrt::Microsoft::UI::Windowing::AppWindow::Create();
+    appWindow.Title(L"Sample-App-Fabric");
```

Comment:
Title shouldn't be set here (arguably also the starting size). This is a case where the title (and starting size) should be set by the app template in-between Build() and Start().

Reply:
But it shouldn't matter I have checked that the title can be changed any time before Start is called and it still reflects the right title.

Reply:
Yes, but the app builder is for all apps, not just "Sample-App-Fabric". Either remove the line or put some better default title string

Reply:
Maybe this would be a better string "SampleFabricApplication"